ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -342,10 +347,14 @@ private void createBanner() {
             return;
 
         mVungleManager.cleanUpBanner(mPlacementForPlay);
+        RelativeLayout.LayoutParams adParams = new RelativeLayout.LayoutParams(RelativeLayout.LayoutParams.WRAP_CONTENT,
+                RelativeLayout.LayoutParams.WRAP_CONTENT);
+        adParams.addRule(RelativeLayout.CENTER_IN_PARENT, RelativeLayout.TRUE);

```

Comment:
@HuFangshuai making sure if we need this, will the parent adLayout be bigger in size than the child view in case of VungleBanner (smart/adaptive) or MREC, or will it be of the same size

Reply:
Yes, we need this. If we don't add this rule, vungle ad located on top left in adLayout by default.
AdMob AdView measure and layout our banner view centered if we returned original view(adLayout with WRAP_CONTENT), so if we return resized view (adLayout with width and height), we must ensure our ad view getting centered in adLayout.